src/models/data/organizationData.ts

```diff
@@ -17,11 +17,14 @@ export class OrganizationData {
     use2fa: boolean;
     useApi: boolean;
     useBusinessPortal: boolean;
+    useSso: boolean;
     selfHost: boolean;
     usersGetPremium: boolean;
     seats: number;
     maxCollections: number;
     maxStorageGb?: number;
+    ssoBound: boolean;
```

Comment:
@cscharf addressing your concerns on the old PR about ssoBound:

I'd agree that it may not be appropriate to have ssoBound on this model as it is currently named, but the inheritance hierarchy implied by these constructors starts with a parent model: ProfileOrganizationResponse. The version of this model on the server already had ssoBound, and it pulls from a table that includes org and orgUser information. The organizationData model is also what is put in storage, which is what is referenced on the organizations page that is relevant to ssoBound. 

I'd argue that instead of removing the property from this model, maybe the model should be renamed to more appropriately contextualize the data it provides. I imagine most of the time an Organization appears in a UI it is going to be more relevant to an OrgUser link than to just an Organization.

Let me know your thoughts on this.

Reply:
Ah, makes sense, thanks for the clarification @addisonbeck , this should be good as-is for now.